### PR TITLE
docs: add context explainer

### PR DESCRIPTION
**Pull request by jrhender**

### CODE REVIEW

File: apps/vc-api/README.md [hunk 1]
```diff
@@ -9,11 +9,16 @@ This [vc-api app](./apps/vc-api) is a NestJs implementation of the [W3C Credenti
...
 
-## Credentials Exchanges
+## Credential Exchanges
 
 Credential exchanges are the processes by which credentials are moved between wallet/agent and issuer/verifer.
 For more information on these processes, see the [exchanges documentation](./docs/exchanges.md). 
 
+## Credential JSON-LD Contexts
+
+The Verifiable Credentials specification used context to ensure that the meanings of terms in a credential are shared by all parties.
```
Comment (whitneypurdum) [inline apps/vc-api/README.md:19]: 'uses' or 'used'?

File: apps/vc-api/docs/contexts.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+ 
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+# Verificable Credentials Contexts
+
+## Contexts in the VC Specification
+
+See the [Contexts section](w3.org/TR/vc-data-model/#contexts) of the specification for more generation information about context in VCs.
```
Comment (Harasz) [inline apps/vc-api/docs/contexts.md:22]:
```suggestion
See the [Contexts section](https://www.w3.org/TR/vc-data-model/#contexts) of the specification for more generation information about context in VCs.
```

File: apps/vc-api/docs/contexts.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+## Loading Contexts in Credentials
+
+Though contexts are indentified by URLs, for security and privacy it is best to load contexts from local copies.
+DIDKit, the SSI library which this implementation of VC-API uses, only supports contexts which are built into library.
+For more information on this, see the (DIDKit documentation on document loading[https://spruceid.dev/docs/didkit/document-loading/].
```
Comment (Harasz) [inline apps/vc-api/docs/contexts.md:28]:
```suggestion
For more information on this, see the [DIDKit documentation on document loading](https://spruceid.dev/docs/didkit/document-loading/).
```

File: apps/vc-api/docs/contexts.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+Given the need to re-build DIDKit in order to add a new context, it is not possible with this current implementation of VC-API
+to issue or verify a credential which refers a context which has is not in the DIDKit [default contexts](https://github.com/spruceid/ssi/tree/main/contexts).
+However, it is possible to use a new context by including the context inline into the credential.
+
+### In-line contexts
+
+Rather than referencing a context via a URI in a JSON-LD `@context` array, it is possible to include the context directly in-line.
+
+For example, a veriable credential may contain a `@context` array as follows:
```
Comment (Harasz) [inline apps/vc-api/docs/contexts.md:38]:
```suggestion
For example, a verifiable credential may contain a `@context` array as follows:
```

File: apps/vc-api/docs/contexts.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+   ],
+   "credentialSubject":{
+      "id":"did:example:123",
+      "customObject": {
+        "customField": "customValue"
+      }
+   },
+   "issuer":"did:example:456",
+   "issuanceDate":"2022-03-18T08:57:32.477Z"
+}
```
Comment (Harasz) [inline apps/vc-api/docs/contexts.md:64]:
```suggestion
   "@context": [
      "https://www.w3.org/2018/credentials/v1",
      {
         "custom": "https://example.com/custom-context-2022#",
         "customObject": {
            "@id": "custom:customObject",
            "@type": "custom:ExampleObject"
         },
         "customField": "custom:customField"
      }
   ],
   "id": "urn:uuid:7f94d397-3e70-4a43-945e-1a13069e636f",
   "type": [
      "VerifiableCredential"
   ],
   "credentialSubject":{
      "id": "did:example:123",
      "customObject": {
        "customField": "customValue"
      }
   },
   "issuer": "did:example:456",
   "issuanceDate": "2022-03-18T08:57:32.477Z"
}
```

**Review by Harasz — CHANGES_REQUESTED** [review]

File: apps/vc-api/docs/contexts.md (hunk 2 above)
Comment (JGiter) [reply apps/vc-api/docs/contexts.md:22]: general?

File: apps/vc-api/docs/contexts.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+See the [Contexts section](w3.org/TR/vc-data-model/#contexts) of the specification for more generation information about context in VCs.
+
+## Loading Contexts in Credentials
+
+Though contexts are indentified by URLs, for security and privacy it is best to load contexts from local copies.
+DIDKit, the SSI library which this implementation of VC-API uses, only supports contexts which are built into library.
+For more information on this, see the (DIDKit documentation on document loading[https://spruceid.dev/docs/didkit/document-loading/].
+
+Given the need to re-build DIDKit in order to add a new context, it is not possible with this current implementation of VC-API
+to issue or verify a credential which refers a context which has is not in the DIDKit [default contexts](https://github.com/spruceid/ssi/tree/main/contexts).
```
Comment (JGiter) [inline apps/vc-api/docs/contexts.md:31]:
```suggestion
to issue or verify a credential which refers a context which is not in the DIDKit [default contexts](https://github.com/spruceid/ssi/tree/main/contexts).
```

**Review by JGiter — APPROVED** [review]

**Pull request merged**